app/controllers/notifications_controller.rb

```diff
@@ -0,0 +1,15 @@
+class NotificationsController < ApplicationController
+  def index
+    @notifications = current_user.notifications
+  end
+
+  def update
+    notification = Notification.find(params[:id])
```

Comment:
Sorry I missed this on the first pass through, can we scope this find to the current users notifications please?